### DISCUSSION TITLE
No Custom Custom Field under Admin interfae

### DISCUSSION THREAD

**Discussion by weezter** [netbox-community/netbox-docker#590] [Getting Help] [answered]
Hi, I just followed the guide to run up netbox with docker.
Everything works but under admin and EXTRAS so can i only see Job results.
I missing Custom Fields.. Any one know why?

**Comment by tobiasge** [comment, marked as answer]
The custom fields are in the main UI in the menu item "Other".

**Reply by manderio** [reply]
I cannot find this, is this still relevant?

**Reply by kkthxbye-code** [reply]
Custom fields are in the CUSTOMIZATION menu in recent versions of netbox.

**Reply by manderio** [reply]
Thanks, however, note that I am using the docker version, and I cannot seem to find any of the fields, note I have gone to admin->ctrl+f(customization) (both with z and s)

Still nothing found. Is this a docker container problem or is this a me problem?

**Reply by kkthxbye-code** [reply]
They are not in the admin interface, they are in the main sidebar in the netbox ui.

**Reply by manderio** [reply]
Thank you so much!
